src/Microsoft.ComponentDetection.Detectors/maven/MavenParsingUtilities.cs

```diff
@@ -37,16 +39,20 @@ private static (string groupId, string artifactId, string version, bool? isDevel
                 results = new[] { results[0], results[1], results[2], results[4], results[5] };
             }
 
+            // 'Compile' is a default scope for maven dependencies.
+            DependencyScope dependencyScope = DependencyScope.Compile;
             var groupId = results[0];
             var artifactId = results[1];
             var version = results[3];
             bool? isDevDependency = null;
             if (results.Length == 5)
             {
                 isDevDependency = string.Equals(results[4], "test", StringComparison.OrdinalIgnoreCase);
+                dependencyScope = Enum.TryParse<DependencyScope>(results[4], true, out dependencyScope) ? dependencyScope : 
```

Comment:
Will this throw when scope is not specified or only if it doesn't match the enum?

Reply:
this will only throw when scope doesn't match the enum. 
if the scope is not specified, `result.Length == 5`  (line # 48) will be false, and default compile scope will be considered.